RELEASE-NOTES.md

```diff
@@ -1,5 +1,91 @@
 # Prefect Release Notes
 
+## Release 2.12.0
+
+### Introducing `Flow.serve()`
+
+We're excited to introduce a radically simple way to deploy flows. 
+
+The new `.serve()` method available on every flow allows you to take your existing flows and schedule or trigger runs via the Prefect UI and CLI. 
+
+This addition makes it easier than it's ever been to get started with Prefect:
+
+```python title="hello.py"
+from prefect import flow
+
+@flow
+def hello(name = "Marvin"):
+    print(f"Hello {name}!")
+
+if __name__ == "__main__":
+    hello.serve(name="hourly-greeting", interval=3600)
```

Comment:
@desertaxle Do we mind being verbose about what `name` and `interval` are here, if this is intended toward new users?

Reply:
At least that `name` is the name of a deployment

Reply:
Added a comment to the example script. Let me know if more clarity would be helpful.